### DISCUSSION TITLE
Not working with Cartflows Extension

### DISCUSSION THREAD

**Discussion by Sash6541** [woocommerce/woocommerce#45047] [WooCommerce, Blocks, and Site Editing]
## What do you like about the Cart & Checkout blocks? 

Clean and well laid out

## What do you think is missing from the Cart & Checkout blocks? 

seems to cause a problem with cartflows order bumps and upsells. Re-directs user to the generic thank you page template

**Comment by nerrad** [comment]
HI there @Sash6541  👋🏻

Thanks so much for taking the time to leave some feedback - much appreciated! We make note of all feedback we receive and we also close issues with the feedback as active issues are used for bugs or feature/enhancements. It does not mean that we don't note or review the feedback. Closed issues can also still be commented on.

The Cart and Checkout blocks are a new change to the WooCommerce checkout flow and extensions will have to update to integrate fully with them. So it is expected that some extensions might not be fully compatible with the blocks just yet. We are currently working on introducing more ways for extension developers to integrate with the new checkout flows and definitely will be encouraging extension developers to integrate their products as those mechanisms become available. 

If you haven't already, it'd be a good idea to let the Cartflows product team know about your experience so it's on their radar as a need from their users (although I think it already is on their radar).
